### PR TITLE
Create general folder structure   bot 2

### LINKED ISSUE #2: create general folder structure - bot
DOD:

- [x] learn the general application diagram 
- [x] decide on folder structure for the project
- [x] make sure that the names of the folders are short and descriptive
- [x] make sure that all team memebrs understand what kind of files goes into which folder

[telegram bot.pdf](https://github.com/user-attachments/files/16322321/telegram.bot.pdf)

### PR DESCRIPTION
**Pull request by hasanMasalha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
 # code-genie-bot
 telegram bot for code genie
+
+
+
+## the structure of the bot 
+
+├── code-genie-bot \
+│  # Package initia lization \
```
Comment (yishain11) [inline README.md:8]: fix space here

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+│  # Main bot entry point \
+│ ├── handlers/ \
+│ │  # Handlers package initialization \
+│ │  # Handle /start command \
```
Comment (yishain11) [inline README.md]: no need to specify each command. just say: handle bot commands, or something similar.
the idea is to help anyone reading the code what kind of files will be found in this dir

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (hasanMasalha) [reply README.md]: done :)

**Review by yishain11 — CHANGES_REQUESTED** [review]
change to pr into dev, not main

**Pull request closed without merging**